src/accountScreen/accountScreen.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import './accountScreen.css'
+import SearchBar from 'material-ui-search-bar'
+import {Card, CardActions, CardHeader, CardText} from 'material-ui/Card';
+
+
+
+const ChatCard = function(props) {
+  if (props.selected) {
+    return (
+      <Card style={{
+        backgroundColor:'#cacaca'}}>
+        <CardHeader
+          title={props.title}
+          avatar={props.avatar}
+        />
+        <CardText>
+          {props.text}
+        </CardText>
+      </Card>        
+    );
+  }
+
+  else {

```

Comment:
Do not add a new line between the closing if tag and the start of the else tag. This code is not readable

Reply:
Noted. Done. Added prettier as well in latest push